lighthouse-core/test/results/sample_v2.json

```diff
@@ -1388,9 +1388,8 @@
           {
             "resourceType": "third-party",
             "label": "Third-party",
-            "requestCount": 2,
```

Comment:
unexpected that our requestCount changes here right shouldn't the default state after these changes be a noop?